### DISCUSSION TITLE
Improvement: Calendar time selection

### DISCUSSION THREAD

**Discussion by Tomdrouv1** [primefaces/community#3089] [PrimeNG]
Hello,

I'm using PrimeNG for my Angular projects for a while now but I always needed to use another library for time picker because the Calendar component doesn't offer a great user experience, in my opinion.

Here's an example of what we could have :

![Screenshot 2025-03-06 144613](https://github.com/user-attachments/assets/10d15255-f1f3-4e3f-aed7-3540ab2658d7)

**Comment by MightyFo0l** [comment]
Would be really great to also add for dates. We build something like this in the past
![afbeelding](https://github.com/user-attachments/assets/91672247-d62b-4ac1-807a-de2e45a27069)

**Reply by Tomdrouv1** [reply]
Indeed it could be nice to have, I'll work on an example
